### PR TITLE
[doc] add gaocegege's introduction.

### PR DESCRIPTION
**Pull request by gaocegege**
add the introduction to gaocegege to source.

PTAL @arrowrowe @at15

### CODE REVIEW

File: source/about.gaocegege.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+type: page
+title: "关于 gaocegege"
+author: gaocegege
+
+---
+
+## gaocegege
+
+MOS 组现任组长，关注的方向比较偏 Ops。其中有 [docker/docker](https://github.com/docker/docker) 和 [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) 等。
```
Comment (at15) [inline source/about.gaocegege.md:9]: what about cyclone?

**Comment by at15** [comment]
吼啊 Though I think the branch name would be better in `author/gaocege`

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/3/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

**Comment by gaocegege** [comment]
Yeah, you are right.

nc17 63s 4av 6f jxe97hq

**Comment by arrowrowe** [comment]
:sheep:

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/3/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

File: source/about.gaocegege.md (hunk 1 above)
Comment (gaocegege) [reply source/about.gaocegege.md:9]: I quit the cyclone dev team now, so I don't mention it in the intro.

**Comment by gaocegege** [comment]
:sheep:

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/3/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

**Pull request merged**